Add flowchart about the status ownership and mention "running" around the absent status (#531)
kernelci/kcidb: doc/submitter_guide.md

decided not to run this specific test, because it was inapplicable.

Finally, the last line in the table above (without a status) corresponds to

> the absent "status" field, and means that the test is either scheduled to
> execute, the system hasn't got to it yet, and so we have no data on anything, or it is still running.

Reviewer: Perhaps:
```suggestion
the absent "status" field, and means that the test is either only scheduled, or is still executing,
and so we have no status data yet.
```
